src/fmu/sumo/explorer/objects/_search_context.py

```diff
@@ -1640,7 +1640,9 @@ def batch_aggregate(self, columns=None, operation=None, no_wait=False):
         """
         assert operation == "collection"
         assert type(columns) is list and len(columns) > 0
-        columns = self.match_field_values("data.spec.columns.keyword", columns)
```

Comment:
Are the `match_field_values` methods used anywhere else or can they be removed completely?

Reply:
Right 👍